backend/src/API/teamAPI.ts

```diff
@@ -1,6 +1,6 @@
 import { v4 as uuidv4 } from 'uuid';
 import PermissionsManager from '../utils/permissionsManager';
-import { Team } from '../DataTypes';
+import { Team } from '../types/DataTypes.d';
```

Comment:
I think the import should be fine without the `.d` extension so just `../types/DataTypes`

Reply:
@riyajaggi should be fixd